tests/InputFiles/CurvedScalarWave/WorldtubeKerrSchild.yaml

```diff
@@ -147,14 +146,11 @@ Worldtube:
         Offset: 0
 
 InterpolationTargets:
-  PsiAlongAxis1:
-    Begin: [0., 0., 0.]
-    End: [0., 0., 400.]
-    NumberOfPoints: 1000
-  PsiAlongAxis2:
-    Begin: [0., 0., 0.]
-    End: [400., 0., 0.]
-    NumberOfPoints: 1000
+  Spheres:
+    LMax: 10
+    Center: [0.,0.,0.]
+    Radius: [300., 400.]
```

Comment:
So with the `ObserveSurfaceData` callback, it can only handle a single radius at the moment. I'm not sure what happens if you use multiple radii. I don't think it's too much work to allow it to handle multiple radii, I just haven't taken the time to do it. So in the meantime, choose only one radius.

Reply:
Hm, perhaps this is related to the weird error I have seen where it would just not write to file sometimes.